task/03-date-tasks.js

```diff
@@ -76,7 +87,22 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+	var TS = Math.abs(startDate.getTime()-endDate.getTime());
+	 var ms = TS%1000;
+	  if (ms < 10) var msstr = '00' + ms; 
+	  else{
+		  if (ms<100) var msstr = '0' + ms ;
+		  else var msstr = ms;
+	  }
+	var s = (TS - ms)/1000;
+	 var so = s%60;
+	  if (so>9) var sostr = so; else var sostr = '0'+so;
+	var m = (s - so)/60;
+	 var mo = m%60;
+	  if (mo>9) var mostr = mo; else mostr = '0'+mo;
+	var h = (m - mo)/60;
+	if (h<=9) return ['0'+h, mostr, sostr].join(":") + "." + msstr 
+	else return [h, mostr, sostr].join(":") + "." + msstr
 }
```

Comment:
`var` is deprecated. It has no sense to use for browsers code even currently.
code style